From c160ee6f09940be5ccffb30585e252190beb5946 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Tue, 29 Jan 2013 14:54:08 +0800
Subject: [PATCH] Revert "rk3168_86v: Basic functions is ok"

This reverts commit aa932b765305b1cee6bfe648e19a0c1a7cd3f0c8.
For fix Makefile, drivers/video/display/Kconfig
---
 Makefile                             | 4 ++--
 drivers/video/display/screen/Kconfig | 3 ++-
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 0ec77bf9bd30..b22d80e944c8 100644
--- a/Makefile
+++ b/Makefile
@@ -194,8 +194,8 @@ SUBARCH := $(shell uname -m | sed -e s/i.86/i386/ -e s/sun4u/sparc64/ \
 export KBUILD_BUILDHOST := $(SUBARCH)
 #ARCH		?= $(SUBARCH)
 ARCH		?= arm
-ifneq ($(wildcard ../../../toolchain/arm-eabi-4.4.3),)
-CROSS_COMPILE	?= ../../../toolchain/arm-eabi-4.4.3/bin/arm-eabi-
+ifneq ($(wildcard ../toolchain/arm-eabi-4.4.3),)
+CROSS_COMPILE	?= ../toolchain/arm-eabi-4.4.3/bin/arm-eabi-
 endif
 ifneq ($(wildcard ../prebuilt/linux-x86/toolchain/arm-eabi-4.4.3),)
 CROSS_COMPILE	?= ../prebuilt/linux-x86/toolchain/arm-eabi-4.4.3/bin/arm-eabi-
diff --git a/drivers/video/display/screen/Kconfig b/drivers/video/display/screen/Kconfig
index e8dc1c762e14..182a360105e5 100755
--- a/drivers/video/display/screen/Kconfig
+++ b/drivers/video/display/screen/Kconfig
@@ -99,9 +99,10 @@ config LCD_RK2928_A720
        bool "RK2928 A720 panel 800x480"
 config LCD_RK2926_V86
 	bool "RK2926 v86 panel 800x480"
+
 config LCD_RK3168_86V
 	bool "RK3168 86v panel 800x480"
-config LCD_HJ080NA
+
 config LCD_HJ080NA
 	bool "HJ080NA_4J 1024X768"
 
-- 
2.35.3

